src/components/file/File.js

```diff
@@ -509,7 +515,15 @@ export class FileComponent extends BaseComponent {
     }
     fileService.downloadFile(fileInfo).then((file) => {
       if (file) {
-        window.open(file.url, '_blank');
+        // commented becouse this way is not supported any more by chrome due to security practices.
+        //window.open(file.url, '_blank');
```

Comment:
We need this to remain, and only execute the code below if ```window.open``` fails. Please see if this works for your use case.

```js
try {
  window.open(file.url, '_blank');
}
catch (err) {
  var hiddenElement = document.createElement('a');
  hiddenElement.href = 'data:' + file.type + ';base64,' + encodeURI(file.data);
  hiddenElement.target = '_blank';
  hiddenElement.download = file.originalName;
  hiddenElement.click();
}
```

Reply:
unfortunately it didn't work because Chrome consider this as a security issue rather than an exception occurred. So the code didn't continue to the fallback branch.

Reply:
@travist I just updated the pull request in away that the new code will run only if file upload mode is in base64. Otherwise, window.open() will work as is.